walrus/database.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, *args, **kwargs):
             scripts.
         """
         script_dir = kwargs.pop('script_dir', None)
+        self.encoding = kwargs.get('encoding') or 'utf-8'
```

Comment:
It looks like redis-py already supports a `charset` parameter:

https://redis-py.readthedocs.org/en/latest/index.html#redis.Redis

Perhaps that should be used instead?